app/forms/shared_ownership_housing_costs_form.rb

```diff
@@ -0,0 +1,45 @@
+class SharedOwnershipHousingCostsForm < BaseHousingCostsForm
+  ATTRIBUTES = (BaseHousingCostsForm::ATTRIBUTES + %i[mortgage mortgage_frequency rent rent_frequency]).freeze
```

Comment:
@willc-work I think you need to add the attribute shared_ownership_mortgage and remove the attribute mortgage here.